src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -127,5 +127,9 @@
             <argument type="service" id="service_container" />
             <argument type="tagged_iterator" tag="container.env_var_loader" />
         </service>
+
+        <service id="monolog.http_client" class="Symfony\Contracts\HttpClient\HttpClientInterface">
+            <factory class="Symfony\Component\HttpClient\HttpClient" method="create" />
```

Comment:
the service could have the `http_client.client` tag so that it can be traced in the profiler, wdyt?

Reply:
I already tested it, and It's a bad idea. It reports N requests and it is misleading when **you** don't trigger HTTP Request because you expect 0. Even if you trigger HTTP Request, you need to open the panel to know how many you did because there are extra request...